### PR TITLE
calendar

### PR DESCRIPTION
**Pull request by SerhiiShapovalenko**
[DEMO LINK](https://SergeyShapovalenko.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+  "mon" 0, 
+  "tue" 1, 
+  "wed" 2, 
+  "thu" 3, 
+  "fri" 4, 
+  "sat" 5,
+  "sun" 6;
+$square-size: 100px;
+$gap: 1px;
+$calendar-windt: ($square-size + $gap) * 7;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/main.scss]:
```suggestion
$calendar-width: ($square-size + $gap) * 7;
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    
+    transition: 0.5s;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/main.scss]: Implement checkpoint
image

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Hi, demo link is not working, try to fix it (did you deploy your project?). And if you have any questions - feel free to ask them in the Slack
image

And what about to split your styles into separated files like reset.scss, card,scss, so on...

File: src/styles/blocks/_calendar.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      }
+    }
+
+    &__day {
+      display: flex;
+      justify-content: $center;
+      align-items: $center;
+      text-align: $center;
+    
+      transition: 0.5s;
```
Comment (loralevitska) [inline src/styles/blocks/_calendar.scss]: Fix previous comment: add properties (transform, background-color)

File: src/styles/blocks/_calendar.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+      &:hover {
+        cursor: pointer;
+        background-color: pink;
+      
+        transform: translate(0, -20px);
+      }
+
+      @include square ($square-size);
+      background-color: #eee;
```
Comment (loralevitska) [inline src/styles/blocks/_calendar.scss]: It is a good practice to make all colors variable

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Let's improve 
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Review by ivbutsykin — APPROVED** [review]
Good job!

**Pull request closed without merging**